File: 07_ServerlessFargatePipeline/python/ServerlessFargatePipeline/fargate_stage.py
from constructs import Construct
from aws_cdk import (
    Stage
)
from .fargate_stack import FargateStack

class FargateStage(Stage):

    def __init__(self, scope: Construct, id: str, **kwargs):
        super().__init__(scope, id, **kwargs)

        service = FargateStack(self, 'FargateStack')

File: 07_ServerlessFargatePipeline/typescript/lib/fargate-stage.ts
import { FargateStack } from './fargate-stack';
import { Stage, StageProps } from 'aws-cdk-lib';
import { Construct } from 'constructs';

export class FargateStage extends Stage {
    constructor(scope: Construct, id: string, props?: StageProps) {
        super(scope, id, props);

        new FargateStack(this, 'Fargate');
    }
}
